server/src/server.ts

```diff
@@ -383,11 +385,49 @@ function onMessage(msg: m.Message) {
         // error: code and message set in case an exception happens during the definition request.
       };
       send(definitionResponse);
+    } else if (msg.method === p.RenameRequest.method) {
+      // https://microsoft.github.io/language-server-protocol/specifications/specification-current/#textDocument_rename
+      let params = msg.params as p.RenameParams;
```

Comment:
There's a bit of code duplication with the rename command.
Maybe it makes sense to move out functionality that could lead to errors in future. E.g. definitely the way the `references` command is called, which parameters, etc. So that a future edit does not change only one place out of two, by mistake.